integration_tests/mockApis/auth.ts

```diff
@@ -153,6 +153,8 @@ const stubUserRoles = () =>
     },
   })
 
+export const mockedUsername = 'USER1'
+
```

Comment:
This could probably go in the default `export` below (and be called with `auth.mockedUsername`, but I'm not too bothered either way